docs/sources/whatsnew/whats-new-in-v9-4.md

```diff
@@ -254,12 +254,6 @@ We've added Discord as a contact point receiver for Grafana Cloud alert rules.
 
 We've made the following changes to alert administration.
 
-#### Better guidance to configure your Alertmanagers
-
-Get additional help while configuring your Alertmanager. If you enter an invalid Alertmanager configuration, an error message displays, and you can choose from a previous working configuration to restart it.
-
-{{< figure src="/media/docs/alerting/alertmanager-config.png" max-width="750px" caption="Better guidance to configure your Alertmanager" >}}
```

Comment:
We could also probably drop the image since nothing references it any more?

Reply:
I thought so, but in a month or so when we want to revive this section we'll gonna need it, and it's easier to have it already there.

Reply:
Sure, but it's in version control so we can always recover it from the git history. No need to have dead objects in main.

But, it's merged now so probably not a big deal.